common/autoconf/generated-configure.sh

```diff
@@ -1100,6 +1100,7 @@ infodir
 docdir
 oldincludedir
 includedir
+runstatedir
```

Comment:
what is this runstatedir thing? Maybe I missed it, but it doesn't seem to be used by the other changes in this PR?

Reply:
I'm not sure why anything in this file changed other than the timestamp which is the only change in the new commit pushed up.